packages/gatsby/src/redux/run-sift.js

```diff
@@ -41,10 +41,9 @@ const prepareQueryArgs = (filterFields = {}) =>
   }, {})
 
 const getFilters = filters =>
-  Object.keys(filters).reduce(
-    (acc, key) => acc.push({ [key]: filters[key] }) && acc,
-    []
-  )
+  Object.keys(filters).map(key => {
+    return { [key]: filters[key] }
+  })
```

Comment:
Nice found!

Reply:
Hm interesting. Is there a lint rule that changes `foo => ({x})` to `foo => { return {x} }` ? Cause that's what I had.

I'm not against this, just a little surprised :) I feel a little guilty for slipstreaming this one in but at the same time, shrug.